clang/test/CIR/Lowering/bitfields.c

```diff
@@ -0,0 +1,32 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-llvm %s -o %t.ll
```

Comment:
Please remove this file. It needs OGCG checks added, and it's probably not adding anything to existing tests.